Changes

```diff
@@ -68,6 +68,11 @@ _______________
 
 ### Bug fixes:
 
+- #11040: Silences false data race observed between caml_shared_try_alloc
+  and oldify. Introduces macros to call tsan annotations which help annotate
+  a ``happens before'' relationship.
+  (Olivier Nicole, Hari Hara Naveen S)
```

Comment:
(If you are still amending the PR, I think you could list me as a reviewer in the Changes entry.)